quickwit-indexing/src/actors/packager.rs

```diff
@@ -276,6 +276,12 @@ fn try_extract_terms(
                 FieldType::F64(_) => {
                     tantivy::u64_to_f64(u64_from_term_data(term_data)?).to_string()
                 }
+                FieldType::Bool(_) => match u64_from_term_data(term_data)? {
+                    0 => false,
+                    1 => true,
+                    _ => bail!("Invalid boolean value"),
+                }
+                .to_string(),
```

Comment:
No optimization here please. This conversion takes nanoseconds and there can only be two values for booleans.

Can you add a unit test to check that the query side of tags?
We have code somewhere that takes a user queyr and extract a TagFilterAST out of it. We need to make sure that the strings produced are matching the tag string you have here.

Maybe it would be nice to be lenient to and allow: `myboolfield:true` and `myboolfield:1` on the query side.

Reply:
Performance gain here is non-sense. The packager takes several seconds to run. This conversion takes  nanoseconds.

Can you add a unit test to make sure that using boolean tags is working?
We have some code that takes a query and extract a predicate on tags based on that query.

Reply:
the lenient thing is nice, @boraarslan you can open an issue on tantivy side to not forget it

Reply:
@fmassot it is more complicated than it might seem unfortunately.

I opened an issue to explain the key issue.
https://github.com/quickwit-oss/quickwit/issues/1655